.github/scripts/trymerge.py

```diff
@@ -1174,16 +1195,22 @@ def merge(pr_num: int, repo: GitRepo,
     org, project = repo.gh_owner_and_name()
     pr = GitHubPR(org, project, pr_num)
     initial_commit_sha = pr.last_commit()['oid']
+    explainer = TryMergeExplainer(force, on_green, land_checks, pr, org, project)
+    on_green, land_checks = explainer.get_flags()
+
     check_for_sev(org, project, force)
+
+    if land_checks:
+        land_check_commit = pr.create_land_time_check_branch(repo, 'viable/strict', force=force, comment_id=comment_id)
+
+    explainer.print_merge_message(land_check_commit, dry_run)
+
     if force or can_skip_internal_checks(pr, comment_id):
         # do not wait for any pending signals if PR is closed as part of co-development process
         return pr.merge_into(repo, dry_run=dry_run, force=force, comment_id=comment_id)
     if (datetime.utcnow() - pr.last_pushed_at()).days > stale_pr_days:
```

Comment:
I think we may want this before creating the land time check branch so that we don't waste resources. We can just do if blah and not force

Reply:
I guess but it adds a bit more code since we need to call print_merge_message twice. land checks and force are mutually exclusive rn (at least in test-infra) so it'll never create a land check branch if force is true.